src/reactjs_jsx_ppx_v4.ml

```diff
@@ -0,0 +1,1455 @@
+("muted", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("name", {jsString = None; typeString = mkPropConst (Lident "string")});
+("nonce", {jsString = None; typeString = mkPropConst (Lident "string")});
+("noValidate", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("open_", {jsString = Some("open"); typeString = mkPropConst (Lident "bool")});
+("optimum", {jsString = None; typeString = mkPropConst (Lident "int")});
+("pattern", {jsString = None; typeString = mkPropConst (Lident "string")});
+("placeholder", {jsString = None; typeString = mkPropConst (Lident "string")});
+("poster", {jsString = None; typeString = mkPropConst (Lident "string")});
+("preload", {jsString = None; typeString = mkPropConst (Lident "string")});
+("radioGroup", {jsString = None; typeString = mkPropConst (Lident "string")});
+("readOnly", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("rel", {jsString = None; typeString = mkPropConst (Lident "string")});
+("required", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("reversed", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("rows", {jsString = None; typeString = mkPropConst (Lident "int")});
+("rowSpan", {jsString = None; typeString = mkPropConst (Lident "int")});
+("sandbox", {jsString = None; typeString = mkPropConst (Lident "string")});
+("scope", {jsString = None; typeString = mkPropConst (Lident "string")});
+("scoped", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("scrolling", {jsString = None; typeString = mkPropConst (Lident "string")});
+("selected", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("shape", {jsString = None; typeString = mkPropConst (Lident "string")});
+("size", {jsString = None; typeString = mkPropConst (Lident "int")});
+("sizes", {jsString = None; typeString = mkPropConst (Lident "string")});
+("span", {jsString = None; typeString = mkPropConst (Lident "int")});
+("src", {jsString = None; typeString = mkPropConst (Lident "string")});
+("srcDoc", {jsString = None; typeString = mkPropConst (Lident "string")});
+("srcLang", {jsString = None; typeString = mkPropConst (Lident "string")});
+("srcSet", {jsString = None; typeString = mkPropConst (Lident "string")});
+("start", {jsString = None; typeString = mkPropConst (Lident "int")});
+("step", {jsString = None; typeString = mkPropConst (Lident "float")});
+("summary", {jsString = None; typeString = mkPropConst (Lident "string")});
+("target", {jsString = None; typeString = mkPropConst (Lident "string")});
+("type_", {jsString = Some("type"); typeString = mkPropConst (Lident "string")});
+("useMap", {jsString = None; typeString = mkPropConst (Lident "string")});
+("value", {jsString = None; typeString = mkPropConst (Lident "string")});
+("width", {jsString = None; typeString = mkPropConst (Lident "string")});
+("wrap", {jsString = None; typeString = mkPropConst (Lident "string")});
```

Comment:
According to [WHATWG HTML living standard](https://html.spec.whatwg.org/#attr-textarea-wrap), this should be an enumerated attribute.
```suggestion
("wrap", {jsString = None; typeString = mkPropVariant ["soft"; "hard"]});
```